lib/api/entities.rb

```diff
@@ -166,6 +171,12 @@ class MergeRequest < ProjectEntity
       expose :milestone, using: Entities::Milestone
     end
 
+    class MergeRequestFiles < MergeRequest
+      expose :diffs, as: :files, using: Entities::RepoDiff do |compare, options|
```

Comment:
Unused block argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.